07_iifo/index.tex

```diff
@@ -1 +1,21 @@
-\chapter{Inter Island Forecasting Organisation (IIFO)}
\ No newline at end of file
+\chapter{Inter Island Forecasting Organisation (IIFO)}
+
+The role of IIFO is to allow islands to make predictions about the likelihood and severity of risks in the event of natural disasters, which correlates to the long term collective risk dilemma, and the returns from foraging, which correlates to the short term collective risk dilemma.
+
+\section{Long Term Collective Risk Dilemma (ltCRD)}
+\label{sec:IIFO:ltCRD}
+
+Islands may build a model to try and predict the likelihood and severity of upcoming disasters. They are then able to share this with the other islands along with a confidence level for their prediction, allowing the island to specify how much they believe their prediction to be the truth so that in the case that the island is wrong, agents might not judge them as harshly if their confidence level was low. On the flipside, a higher confidence level indicates that other islands should potentially place more trust in this prediction with the risk that the island offering the information will be judged more harshly if they turn out to be wrong. 
+
+The motivation behind sharing this information is that with more people contributing to the common pool, the more the easily the effects of the disaster can be mitigated. Sharing predictions allows the islands to better manage their resources as they have a better idea of when they should be donated to the common pool or not.
+
+Inaccurate predictions may mean that the islands are not prepared for a coming disaster, as it was predicted to come much later, meaning the common pool may not yet be large enough to mitigate the disaster effectively. Alternatively, the islands may over prepare and donate too much to the common pool when a disaster is not imminent, meaning they have unnecessarily reduced their own resources making it harder to forage and build up resources again.
```

Comment:
suggested re-writing:

Inaccurate predictions may result in a situation where the islands are not at all prepared for the incoming disaster. **An example of this is to predict an incoming disaster belatedly, meaning that the common pool may not yet be sufficient enough to mitigate the disaster effectively when it hits the archipelago.** **Alternatively, the islands may also overprepare and donate too much to the common pool when, in fact, a disaster is not imminent, meaning they have unnecessarily reduced their own resources which makes it harder for them to forage and build up resources again.**